SS_write_report.tpl

```diff
@@ -2546,7 +2546,7 @@ FUNCTION void write_bigoutput()
               get_growth3(styr,t,s, subseas);
               Make_AgeLength_Key(s, subseas);  //  spawn subseas
             }
-            get_mat_fec();
+            if(WTage_rd==0) get_mat_fec();
```

Comment:
@Rick-Methot-NOAA , can you explain this line to me? Was this a separate issue from the seasonal recruitment or part of the same?